### ISSUE TITLE
Game Crash on Start

### ISSUE THREAD

**Issue by Magalia1020** [ToCraft/craftedcore#5]
### Mod loader

Fabric

### Minecraft version

1.20.4

### Remorphed version

3.3

### Woodwalkers version

3.2

### CraftedCore version

3.0

### Modloader version

Fabric 0.15.7 & Fabric API 0.96.4

### The latest.log file

Can't open gist, please look for the files in Other

### Issue description

Game crashed immediately when I started the game. At first I downloaded the lastest version of crafted core which is 3.1.2, and the game crashed on start when I launched it. My launcher said I need to replace crafted core with version 3.0 instead of 3.1.2 so I did it just as it said, but the game still crashed.

### Steps to reproduce

1. Downloaded the mods with their lastest versions.
2. Game crashed on start. Need to replace crafted core with version 3.0 instead of 3.1.2
3. Uninstalled crafted core 3.12 and installed 3.0
4. Game crashed on start.

### Other

[minecraft-exported-crash-info-2024-03-14T20-36-10.zip](https://github.com/ToCraft/Remorphed/files/14602395/minecraft-exported-crash-info-2024-03-14T20-36-10.zip)

**Comment by ToCraft** [comment]
Alright, I just went through your crash log and it doesn't look like it's crashing because of one of my mods but because of "[CreativeCore](https://github.com/CreativeMD/CreativeCore/issues)" or "[Friends & Foes](https://github.com/Faboslav/friends-and-foes/issues)" instead. Please report the issue to them.

**Comment by Natedogwoooof** [comment]
> Alright, I just went through your crash log and it doesn't look like it's crashing because of one of my mods but because of "[CreativeCore](https://github.com/CreativeMD/CreativeCore/issues)" or "[Friends & Foes](https://github.com/Faboslav/friends-and-foes/issues)" instead. Please report the issue to them.

idk not trying to be mean cuz im having the same problom cuz your mod also crashed enchanting infuser and moonlight lib and createive core and friends and foes

see same crash happens with diffrent mods

**Comment by Natedogwoooof** [comment]
[crash-2024-03-14_17.08.01-client.txt](https://github.com/ToCraft/craftedcore/files/14608445/crash-2024-03-14_17.08.01-client.txt)
[crash-2024-03-14_16.55.32-client.txt](https://github.com/ToCraft/craftedcore/files/14608457/crash-2024-03-14_16.55.32-client.txt)

**Comment by Natedogwoooof** [comment]
2 diffrent mods no rush to fix the bug tho

**Comment by ToCraft** [comment]
Right... I just tested it and found the issue. It's actually a bug implemented by Remorphed. I'll try to publish a minior update with the fix as soon as possible.
